controllers/start.go

```diff
@@ -46,18 +47,31 @@ func StartManager(cfg ControllerConfig) {
 		os.Exit(1)
 	}
 
-	if err = (&ContainerImageBuildReconciler{
+	controller := &ContainerImageBuildReconciler{
 		Log:       ctrl.Log.WithName("controllers").WithName("ContainerImageBuild"),
 		Client:    mgr.GetClient(),
 		Scheme:    mgr.GetScheme(),
 		Recorder:  mgr.GetEventRecorderFor("containerimagebuild-controller"),
 		JobConfig: cfg.JobConfig,
-	}).SetupWithManager(mgr); err != nil {
+	}
+
+	if err = controller.SetupWithManager(mgr); err != nil {
 		setupLog.Error(err, "Unable to create controller", "controller", "ContainerImageBuild")
 		os.Exit(1)
 	}
 	// +kubebuilder:scaffold:builder
 
+	if cfg.GCInterval != 0 {
+		go func() {
+			for {
```

Comment:
T(some time last week)IL about https://golang.org/pkg/time/#Ticker

Reply:
OoOoOoO...you fancy. Let me check it out.

Reply:
https://github.com/dominodatalab/forge/pull/49/commits/71d838a93d5843b5aeb4edd062c3fe63bfb8503d